var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -166,6 +167,14 @@ class Openmpi(AutotoolsPackage):
     patch('configure.patch', when="@1.10.1")
     patch('fix_multidef_pmi_class.patch', when="@2.0.0:2.0.1")
 
+    # Vader Bug: https://github.com/open-mpi/ompi/issues/5375
```

Comment:
ok, but the original patch seems to contain a bit more? Can this be ignored?
https://github.com/open-mpi/ompi/pull/5705/files